game/scripts/vscripts/modifiers/modifier_custom_roshan_attack_speed.ts

```diff
@@ -0,0 +1,18 @@
+import { BaseModifier, registerModifier } from "../lib/dota_ts_adapter";
+
+@registerModifier()
+export class modifier_custom_roshan_attack_speed extends BaseModifier {
```

Comment:
Why is this modifier at all?

Reply:
One of the modifiers I removed due to scaling reasons (either "modifier_roshan_inherent_buffs" or "modifier_roshan_devotion") have an attack speed buff, Rosh attacking looked weird without it since it's like +100 AS, so I added this in to compensate